markdown/plext/using_pljava.html.md.erb

```diff
@@ -2,205 +2,222 @@
+	root@hawq-node$ ldconfig
 	```
 4. Make sure that your HAWQ cluster is running, you have sourced `greenplum_path.sh` and that your `$GPHOME` environment variable is set.
 
+    ``` shell
+	$ ssh gpadmin@master
+	gpadmin@master$ source /usr/local/hawq/greenplum_path.sh
+	gpadmin@master$ hawq state
+	```
 
 ### <a id="enablepljava"></a>Enable PL/Java and Install JAR Files 
 
-To use PL/Java:
+To use PL/Java, you must:
 
 1. Enable the language for each database.
-1. Install user-created JAR files on all HAWQ hosts.
-1. Add the names of the JAR files to the HAWQ `pljava_classpath` server configuration parameter. This parameter value should identify a list of the installed JAR files.
+1. Install custom JAR files on all HAWQ hosts.
+1. Add the names of the JAR files to the HAWQ `pljava_classpath` server configuration parameter. This parameter value should identify a list of all installed JAR files.
 
 #### <a id="enablepljava"></a>Enable PL/Java and Install JAR Files 
 
 Perform the following steps as the `gpadmin` user:
 
-1. Enable PL/Java by running the `$GPHOME/share/postgresql/pljava/install.sql` SQL script in the databases that will use PL/Java. The `install.sql` script registers both the trusted and untrusted PL/Java languages. For example, the following command enables PL/Java on a database named `testdb`:
+1. Enable PL/Java by running the `$GPHOME/share/postgresql/pljava/install.sql` SQL script on the databases that will use PL/Java. The `install.sql` script registers both the trusted (`java`) and untrusted PL/Java (`javau`) languages. For example, the following command enables the PL/Java languages on a database named `testdb`:
 
 	``` shell
-	$ psql -d testdb -f $GPHOME/share/postgresql/pljava/install.sql
+	gpadmin@master$ psql -d testdb -f $GPHOME/share/postgresql/pljava/install.sql
 	```
 	
 	To enable the PL/Java extension in all new HAWQ databases, run the script on the `template1` database: 
 
     ``` shell
-    $ psql -d template1 -f $GPHOME/share/postgresql/pljava/install.sql
+    gpadmin@master$ psql -d template1 -f $GPHOME/share/postgresql/pljava/install.sql
     ```
 
-    Use this option *only* if you are certain you want to enable PL/Java in all new databases.
+    Use this option *only* if you are certain you want to enable both the trusted and untrusted versions of PL/Java in all newly-created databases. This operation takes effect on all databases created after the command is invoked.
 	
-2. Copy your Java archives (JAR files) to `$GPHOME/lib/postgresql/java/` on all HAWQ hosts. This example uses the `hawq scp` utility to copy the `myclasses.jar` file located in the current directory:
+2. Copy all custom Java JAR files to `$GPHOME/lib/postgresql/java/` on all HAWQ nodes. This example uses the `hawq scp` utility to copy the `myclasses.jar` file located in the current directory to the list of hosts identified in the `hawq_hosts` file, also located in the current directory:
```

Comment:
I think it would be cleaner to remove "located in the current directly" from the sentence, and just include `./` in the actual command.  The way it reads now suggest that the current directory is kept on the destination hosts, but that's not what was intended.